spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -2313,4 +2313,21 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
       }
     }
   }
+
+  test("ArrayInsert") {
```

Comment:
You can add an `assume` call here to skip the test for older Spark versions.

```suggestion
  test("ArrayInsert") {
    assume(isSpark34Plus)
```